docs/source/raft_configuration.md

```diff
@@ -204,21 +200,11 @@ alive) you are effectively stuck until the original offline node is resurrected.
 
 Adding a new node to a Raft cluster is done by:
 
-  1. **Adding the TLS certificates** of the new node to the channel through a
-  channel configuration update transaction. Note: the new node must be added to
-  the system channel before being added to one or more application channels.
-  2. **Fetching the latest config block** of the system channel from an orderer node
-  that's part of the system channel.
-  3. **Ensuring that the node that will be added is part of the system channel**
-  by checking that the config block that was fetched includes the certificate of
-  (soon to be) added node.
-  4. **Starting the new Raft node** with the path to the config block in the
-  `General.BootstrapFile` configuration parameter.
-  5. **Waiting for the Raft node to replicate the blocks** from existing nodes for
-  all channels its certificates have been added to. After this step has been
-  completed, the node begins servicing the channel.
-  6. **Adding the endpoint** of the newly added Raft node to the channel
-  configuration of all channels.
+  1. **Ensuring the orderer organization that owns the new node is one of the orderer organizations on the channel**. If the orderer organization is not an administrator, the node will be unable to pull blocks as a follower or to be joined to the consenter set.
+  2. **Starting the new ordering node**. If you are using the new process to create and join a channel, you do not need to point to a configuration block when starting the node.
```

Comment:
"new process" is confusing because we don't include the "old" process here. Maybe something more specific like:
When you use the channel management API...
but I don't think we have defined that anywhere either.
or for consistency say  When you use the `osnadmin CLI`